client/core/core.go

```diff
@@ -6539,3 +6539,14 @@ func walletDefinition(assetID uint32, walletType string) (*asset.WalletDefinitio
 	}
 	return walletDef, nil
 }
+
+// GetWalletLogFilePath returns the path to the wallet's log file.
+func (c *Core) GetWalletLogFilePath(assetID uint32) (string, error) {
```

Comment:
I'm interested in your opinion on this, but generally I think it's best to omit "get" function name prefixes unless it's alluding to some kind of actual remote fetch operation.  The getting goes on only in the webserver, so let's nuke the prefix from the Core and xcWallet methods.

Reply:
That makes sense, I agree.